clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1644,7 +1644,7 @@ def load_clinical_csv(clinical_dir: str, filename: str) -> pd.DataFrame:
 
     pattern = filename + r"(_\d{1,2}[A-Za-z]{3}\d{4})?.csv"
     files_matching_pattern = [
-        f for f in Path(clinical_dir).rglob("*.csv") if re.search(pattern, (f.name))
+        f for f in Path(clinical_dir).rglob("*.csv") if re.match(pattern, (f.name))
```

Comment:
I agree with @AliceJoubert. Not listing hidden files and folders will be more explicit than changing search for match.
I believe changing the rglob pattern for `.rglob("[!.]*.csv")` should do the trick.
Would be nice to add some unit tests to make sure this is working as expected, especially if there are csv files in subfolders where both the subfolders and the files could be hidden.